RestructuredText/reStructuredText.sublime-syntax

```diff
@@ -9,44 +9,42 @@ file_extensions:
 scope: text.restructuredtext
 contexts:
   main:
-    - match: '^([ \t]*)(?=\S)'
+    - match: '^(?=(\s*)\S)'
       push:
         - meta_content_scope: meta.paragraph.restructuredtext
-        - match: ^(?!\1(?=\S))
+        - match: '^(?!\1\s+)(?=\s*\S+)'
```

Comment:
A paragraph should end when there is a blank line inbetween, don't you think?

Reply:
Is there any way to test this?

Reply:
Hm, I see now that the old syntax didn't do this either. Probably not actually worth bothering with.

Reply:
I think you're right though, this change was incorrect, I'll revert this line